retryx/retry.go

```diff
@@ -0,0 +1,71 @@
+package retryx
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
+const (
+	DefaultDuration   = 500 * time.Millisecond
+	DefaultMaxRetries = 3
+)
+
+func QuickRetry(fn func() error, opts ...RetryOption) error {
```

Comment:
I would be more explicit in the naming of the function as well as document the behaviour. Same for the one above.

```suggestion
func ContantRetry(fn func() error, opts ...RetryOption) error {
```